docs/usage/administration/admin_api/federation.md

```diff
@@ -0,0 +1,81 @@
+# Federation API
+
+This API allows a server administrator to manage the federation.
+
+**Note**: This API is new, experimental and "subject to change".
```

Comment:
Not sure the quotes make sense here---they look like [scare quotes](https://en.wikipedia.org/wiki/Scare_quotes) to me, but I'm not sure what you refer to.